### PR TITLE
add assignment submission classes

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**

### CODE REVIEW

File: d2l-hm-constants-behavior.html [hunk 1]
```diff
@@ -116,9 +116,15 @@
 			},
 			assignments: {
 				assignment: 'assignment',
+				attachment: 'attachment',
+				attachmentList: 'attachment-list',
 				instructions: 'instructions',
 				file: 'file',
-				link: 'link'
+				latest: 'latest',
+				link: 'link',
```
Comment (ryantmer) [inline d2l-hm-constants-behavior.html]: I take it this is `link` like "this assignment is: view this URL" kind of thing, yeah?

Comment (jstefaniuk-d2l) [reply d2l-hm-constants-behavior.html]: Dunno! That one's pre-existing, although I would assume that it's that (I think there's both file attachments and link attachments for assignments)

Comment (ryantmer) [reply d2l-hm-constants-behavior.html]: riiiight, makes sense! Just caught my eye, all is good 😀

**Comment by jstefaniuk-d2l** [comment]
@CodeBaboon @ryantmer I need a code owner approval apparently

**Comment by jstefaniuk-d2l** [comment]
Especially in the repo with the least rigorous code reviews XD

**Pull request merged**